docs/reference/msbuild-targets.md

```diff
@@ -442,8 +442,12 @@ Additional restore settings may come from MSBuild properties in the project file
 | `NuGetLockFilePath` | A custom location for the lock file. The default location is next to the project and is named `packages.lock.json`. |
 | `RestoreForceEvaluate` | Forces restore to recompute the dependencies and update the lock file without any warning. |
 | `RestorePackagesConfig` | An opt-in switch, that restores projects with packages.config. Support with `MSBuild -t:restore` only. |
+| `RestoreRepositoryPath` | packages.config only. Specifies the packages directory to which the packages should be restored. SolutionDirectory will be used if not specified. |
 | `RestoreUseStaticGraphEvaluation` | An opt-in switch to use static graph MSBuild evaluation instead of the standard evaluation. Static graph evaluation is an experimental feature that's significantly faster for large repos and solutions. |
 
+The `ExcludeRestorePackageImports` property is used by NuGet in PackageReference to ensure that build targets from packages brought in by restore cannot affect the next restore.
+As such, this is not a property that should be modified or set in any MSBuild files.
```

Comment:
Why document this property if users should never use it?

Reply:
Because people do :D 

I know of at least 4-5 instances where people used this property and they were getting inconsistent results.

Reply:
Should we rename this property to something like `_ExcludeRestorePackageImports`?

Reply:
The next time restore is run.

Reply:
> Should we rename this property to something like _ExcludeRestorePackageImports?

Too late for that. It's been used in way too many places.

Reply:
I tweaked it per your suggestion @loic-sharma. Thanks!